configs/components/augeas.rb

```diff
@@ -50,30 +50,19 @@
   end
 
   if platform.is_rpm? && !platform.is_aix?
-    pkg.build_requires 'readline-devel'
-    pkg.build_requires 'pkgconfig'
-
-    if platform.is_cisco_wrlinux?
-      pkg.requires 'libreadline6'
-    else
-      pkg.requires 'readline'
-    end
-
     if platform.architecture =~ /aarch64|ppc64le|s390x/
       pkg.build_requires "runtime-#{settings[:runtime_project]}"
       pkg.environment "PATH", "/opt/pl-build-tools/bin:$(PATH):#{settings[:bindir]}"
       pkg.environment "CFLAGS", settings[:cflags]
       pkg.environment "LDFLAGS", settings[:ldflags]
     end
   elsif platform.is_deb?
-    pkg.build_requires 'libreadline-dev'
     if platform.name =~ /debian-9/
       pkg.requires 'libreadline7'
```

Comment:
Should these just get bounced up to the platform level as well?

Reply:
I believe pkg.requires if for runtime requirements to be added to the package, so I think these need to stay.

Reply:
Now that I think about it, this... actually probably does nothing, since we build the runtime as a tarball and not as a full package?